slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/SlashingProtection.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.slashingprotection;
+
+import org.apache.tuweni.units.bigints.UInt64;
+
+// One of these exists per key
```

Comment:
Is this actually the case? I thought the DbBackedSlashingProtection was used for multiple keys

Reply:
removed